### ISSUE TITLE
Question about the AR model in Video World 2

### ISSUE THREAD

**Issue by Markus-Pobitzer** [bytedance/VideoWorld#16]
Dear Zhongwei Ren,

Thank you for he nice work and the public code. I want to apply VideoWorld 2 on a painting task and going through your code I noticed that the AR model and training code is missing. In the paper it is mentioned that NVIDIA Cosmos AR 4B is used, and in Figure 3 a two-stage training pipeline is shown. From my understanding the training mentioned in the [Readme](https://github.com/ByteDance-Seed/VideoWorld/tree/main/VideoWorld2#start-training) corresponds to the first stage of Figure 3.

Maybe you can confirm this, will the AR training code be released, or is it similar to the code in Video World 1?

**Comment by MaverickRen** [comment]
Thank you very much for your interest! Applying VideoWorld to drawing tasks is a fascinating idea. For simple sketches or step-by-step drawing demonstrations, I believe the latent action/dynamic model is capable of encoding visual changes across frames.
Regarding the code: yes, this repository primarily provides the dLDM training pipeline. The AR model is flexible and can be any autoregressive encoder—you can directly use the AR model from VW 1, or alternatives such as Qwen, AR video generation models, etc.

**Comment by Markus-Pobitzer** [comment]
Thank you for the response.

I did some experiments and the VQ-VAE seems to learn the painting task in a simplified way.
